engines/hypno/detection.cpp

```diff
@@ -0,0 +1,156 @@
+	{"wetlands", "Wetlands"},
+	{"soldier-boyz", "Soldier Boyz"},
+	{0, 0}};
+
+static const ADGameDescription gameDescriptions[] = {
+	{"sinister-six", // Demo from the US release
+	 "Demo",
+	 AD_ENTRY2s("DATA.Z",  "2a9c7cf8920ec794482f0a5873102da5", 1285960,
+				"DCINE1.SMK", "1ff3db09d148e8dd8b56d2e87e7296b8", 493752),
+	 Common::EN_USA,
+	 Common::kPlatformDOS,
+	 ADGF_TESTING | ADGF_DEMO,
+	 GUIO1(GUIO_NOMIDI)},
+	{"sinister-six", // US release
+	 _s("Missing game code"),
+	 AD_ENTRY2s("COMBAT.EXE", "bac1d734f2606dbdd0816dfa7a5cf518", 255115,
+				"SPIDER.EXE", "bac1d734f2606dbdd0816dfa7a5cf518", 248056),
+	 Common::EN_USA,
+	 Common::kPlatformDOS,
+	 ADGF_UNSUPPORTED,
+	 GUIO1(GUIO_NOMIDI)},
+	{"wetlands", // Wetlands Demo Disc (November 1995)
+	 "Demo",
+	 AD_ENTRY3s("wetlands.exe", "15a6b1b3819ef002438df340509b5373", 642231,
+				"wetdemo.exe", "15a6b1b3819ef002438df340509b5373", 458319,
+				"demo.exe", "15a6b1b3819ef002438df340509b5373", 533221),
+	 Common::EN_USA,
+	 Common::kPlatformDOS,
+	 ADGF_TESTING | ADGF_DEMO,
+	 GUIO1(GUIO_NOMIDI)},
+    {"wetlands", // Personal Computer World (UK) (May 1996) - Chapter 11 demo
+	 "Demo",
+	 AD_ENTRY2s("wetlands.exe", "15a6b1b3819ef002438df340509b5373", 553355,
+				"missions.lib", "6ffa658f22a00b6e17d7f920fcc13578", 12469),
+	 Common::EN_GRB,
+	 Common::kPlatformDOS,
+	 ADGF_TESTING | ADGF_DEMO,
+	 GUIO1(GUIO_NOMIDI)},
+     {"wetlands", // PC Gamer Disc 12 (November 1995) - Chapter 31 demo
+	 "Demo",
+	 AD_ENTRY2s("wetlands.exe", "15a6b1b3819ef002438df340509b5373", 553355,
+				"missions.lib", "34b922fac8f64546c0690aa83f09e98e", 40891),
+	 Common::EN_USA,
+	 Common::kPlatformDOS,
+	 ADGF_TESTING | ADGF_DEMO,
+	 GUIO1(GUIO_NOMIDI)},
+
+	{"wetlands", // Wetlands (US)
+	 "",
+	 AD_ENTRY2s("wetlands.exe", "15a6b1b3819ef002438df340509b5373", 647447,
+				"missions.lib", "aeaaa8b26ab17e37f060334a311a3ff6", 309793),
+	 Common::EN_USA,
+	 Common::kPlatformDOS,
+	 ADGF_UNSTABLE,
+	 GUIO1(GUIO_NOMIDI)},
+	{"wetlands", // Wetlands (FR)
+	 "",
+	 AD_ENTRY2s("wetlands.exe", "edc5b0c0caf3d5b01d344cb555d9a085", 629575,
+			    "missions.lib", "aeaaa8b26ab17e37f060334a311a3ff6", 309793),
+	 Common::FR_FRA,
+	 Common::kPlatformDOS,
+	 ADGF_UNSTABLE,
+	 GUIO1(GUIO_NOMIDI)},
+	{"wetlands", // Wetlands (ES)
+	 "",
+	 AD_ENTRY2s("wetlands.exe", "8d0f3630523da827bb25e665b7d3f879", 644055,
+				"missions.lib", "aeaaa8b26ab17e37f060334a311a3ff6", 309793),
+	 Common::ES_ESP,
+	 Common::kPlatformDOS,
+	 ADGF_UNSTABLE,
+	 GUIO1(GUIO_NOMIDI)},
+	{"soldier-boyz", // Solidier Boyz (US)
+	 _s("Missing game code"),
+	 AD_ENTRY2s("boyz.exe", "bac1d734f2606dbdd0816dfa7a5cf518", 263347,
+				"setup.exe", "bac1d734f2606dbdd0816dfa7a5cf518", 160740),
+	 Common::EN_USA,
+	 Common::kPlatformWindows,
+	 ADGF_UNSUPPORTED,
+	 GUIO1(GUIO_NOMIDI)},
+	AD_TABLE_END_MARKER};
+} // End of namespace Hypno
+
+static const char *const directoryGlobs[] = {
+	"boyz",
+	"sixdemo",
+	"demo",
+	"factory",
+	"wetlands",
+	"movie",
+	"c_misc",
+	0};
```

Comment:
We prefer to have the closing curly brackets at the separate line except for the nested tables.

Reply:
Fixed.